auslib/db.py

```diff
@@ -2490,6 +2491,53 @@ def _putWatchdogValue(self, changed_by, value, where=None, transaction=None, dry
             super(Dockerflow, self).update(where=where, what=value, changed_by=changed_by, transaction=transaction, dryrun=dryrun)
 
 
+class EmergencyShutoff(AUSTable):
+    def __init__(self, db, metadata, dialect):
+        self.table = Table('emergency_shutoff', metadata,
+                           Column('shutoff_id', Integer, primary_key=True, autoincrement=True),
+                           Column('product', String(15), nullable=False),
+                           Column('channel', String(75), nullable=False),
+                           Column('updates_disabled', CompatibleBooleanColumn, default=False, nullable=False),
```

Comment:
"not updates_disabled" is a double negative, and makes any code using this tougher to read. "enabled" would be better here, then we'll have "enabled" and "not enabled", which are more readable IMO.